### DISCUSSION TITLE
Support for theme-color meta tag

### DISCUSSION THREAD

**Discussion by dyegoaurelio** [mantinedev/mantine#6137] [Feature requests]
Mantine already has support for setting the correct color scheme before hydration, which is great!

But another great addition to it would be to also set the correct [theme color](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color) before hydration, so mobile browsers also can set it's color quickly and without flashes, just like  it's already happening with the color scheme.

I already figured out how to do it, you can check it out here https://mantine-theme-colors.vercel.app/
(make sure to use a compatible browser, meaning safari on IOS or chrome on Android (you need to have light mode enabled on your device if you're testing it on chrome) )

Would a PR adding this feature the the library be welcome?

**Comment by rtivital** [comment]
It is outside of the library scope. It is not compatible with ssr. It is not planned to be added.
